.github/stale.yml

```diff
@@ -0,0 +1,25 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 120
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned 
+  - area/security
+  - area/tls
+  - area/docs
+  - good first issue
+  - benchmarking
+  - help wanted
+  - epic
+  - rfc
+  - roadmap 
+# Label to use when marking an issue as stale
+staleLabel: wontfix
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed in 14 days if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: true
```

Comment:
From looking at the [probot source](https://github.com/probot/stale/blob/e285e96ff5f3bd6a10202f6179e61b5754740425/lib/schema.js#L31-L33), it appears that `closeComment` needs to be either a comment string or false. As written, I think this will close issues with the message "true". I'd recommend changing to `closeComment: false`.